model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/InfinispanKeycloakTransaction.java

```diff
@@ -155,12 +155,12 @@ public String toString() {
             tasks.put(taskKey, new CacheTaskWithValue<V>(value) {
                 @Override
                 public void execute() {
-                    decorateCache(cache).replace(key, value);
+                    decorateCache(cache).replace(key, value, lifespan, lifespanUnit);
                 }
 
                 @Override
                 public String toString() {
-                    return String.format("CacheTaskWithValue: Operation 'replace' for key %s", key);
+                    return String.format("CacheTaskWithValue: Operation 'replace' for key %s, lifespan %d TimeUnit %s", key, lifespan, lifespanUnit.toString());
```

Comment:
explicit `toString()` is not necessary, `%s` would do this automatically
```suggestion
                    return String.format("CacheTaskWithValue: Operation 'replace' for key %s, lifespan %d TimeUnit %s", key, lifespan, lifespanUnit);
```